api/src/main/java/org/openmrs/validator/UserValidator.java

```diff
@@ -186,11 +190,15 @@ public boolean isUserNameValid(String username) {
 	 * @should return true if email valid
 	 */
 	public boolean isUserNameAsEmailValid(String username) {
```

Comment:
Where do you call this method? You should not have an method as public, validator logic should be internal to a validator